lib/stack.js

```diff
@@ -1,19 +1,25 @@
+const LinkedList = require("./linked-list");
+
 class Stack {
     constructor() {
-        // this.store = ...
-        throw new Error("This method has not been implemented!");
+        this.store = new LinkedList()
     }
 
-    push() {
-        throw new Error("This method has not been implemented!");
+    push(value) {
+        this.store.addFirst(value)
     }
 
     pop() {
-        throw new Error("This method has not been implemented!");
+        if (!this.store.head) return null;
+        const poppedValue = this.store.head.value
+        this.store.head = this.store.head.next
+        return poppedValue
     }
 
     isEmpty() {
-        throw new Error("This method has not been implemented!");
+        if (!this.store.head) return true;
+
+        if (this.store.head.next) return false;
```

Comment:
This could give a false empty when the Stack has 1 element.  You can just return false if the prior if statement doesn't exit. 

```suggestion
        return false;
```